src/components/RightPopup.tsx

```diff
@@ -0,0 +1,188 @@
+import Image from "next/image";
+import React, { FC, useCallback, useState } from "react";
+import { useSetRecoilState } from "recoil";
+import styled from "styled-components";
+
+import Button from "./Button";
+import CreatorPopup from "./PopupContents/CreatorPopup";
+import TermsPopup from "./PopupContents/TermsPopup";
+
+import popupState from "utils/states/popupState";
+import { ellipsisText } from "utils/style/commonStyle";
+
+const TEMP_IMAGE = "/temp_profile.png";
+const TEMP_NAME = "Temp";
+const TEMP_SCHOOL = "서울대학교";
+
+const RightPopup: FC = () => {
+  const [isOpened, setIsOpened] = useState(false);
+  const setPopupList = useSetRecoilState(popupState);
+
+  const handleCreatorShow = useCallback(() => {
```

Comment:
```suggestion
  const handlePopupShow = useCallback((Popup) => {
```
이런식으로 컴포넌트를 인자로 넘겨서 함수는 하나만 정의하는건 어떤가유

Reply:
반영했습니당~